servers/src/main/java/tachyon/master/block/meta/MasterWorkerInfo.java

```diff
@@ -70,6 +70,10 @@ public MasterWorkerInfo(long id, NetAddress address) {
     mBlocks = new HashSet<Long>();
     mToRemoveBlocks = new HashSet<Long>();
     mIsRegistered = false;
+    mTotalBytesOnTiers = Collections.emptyList();
+    mUsedBytesOnTiers = Collections.emptyList();
+    mBlocks = Collections.emptySet();
```

Comment:
Oh wow they already are, sorry about that >.<

I got tripped up by the fields being initialized in a different order from how they were declared, so I've reordered the initializations to the order of declaration.